### ISSUE TITLE
Get an favicon.ico

### ISSUE THREAD

**Issue by andialbrecht** [Arachnid/bloggart#18]
A default favicon.ico is currently missing :)

**Comment by hymerman** [comment]
Yep, I'd like this. Also, I've tried to add my own favicon.ico to the themes/default/static/ directory (and the images directory) and it's not getting picked up, which I think is expected. How would you have the favicon.ico file be specific to a theme?

**Comment by ghost** [comment]
in my local copy i have added oner to the themes/default/static/ , and here is the snipit i used to make it work only in that theme ( i'll create a git fork for all my changes and such soon , still getting used to git )

in themes/default/base.html add the following in the head
<link rel="SHORTCUT ICON" href="{{config.url_prefix}}/static/default/favicon.ico"/>

**Comment by ghost** [comment]
ugh it mangled the html, use this
link rel="SHORTCUT ICON" href="{{config.url_prefix}}/static/default/favicon.ico"

with brackets on either side

**Comment by ghost** [comment]
that code is now in my branch, along with a green-ish "doodle" favicon.ico ( that you can grab directly from here http://www.brandonholtsclaw.com/static/default/favicon.ico )
